### DISCUSSION TITLE
iSpindel - Default Polynomial - Impact Readings?

### DISCUSSION THREAD

**Discussion by higginsg** [universam1/iSpindel#594] [Q&A]
Yesterday I did the sugar water method on two iSpindels that I had working some time ago. I did not change the Polynomial back to the default prior to doing the sugar water calibrations.

Values of Polynomial for two iSpindels while doing the sugar water method:
#1: -0.00031*tilt^2+0.557*tilt-14.054
#2: 0.8357+0.009305*tilt-0.0001545*tilt*tilt+0.000001048*tilt*tilt*tilt

My new readings for each iSpindel after doing the sugar water calibrations:
#1: 0.8671+0.007988*tilt-0.0001382*tilt*tilt+9.9453*tilt*tilt*tilt
#2: 0.8523+0.009463*tilt-0.0001754*tilt*tilt+0.000001272*tilt*tilt*tilt

I noticed that #1 has a 9.9453 at the end of the new reading prior to the *tilt*tilt*tilt and #2's new readings are way off with all the zeros. I also put Spindel #1 into my wort and it's reading at 1.010 while my hydrometer has the wort at 1.055 (it's reading wrong including the tilt value).

My questions:
1. Was I supposed to put the default polynomial back in the iSpindel prior to doing the sugar water calibrations of -0.00031*tilt^2+0.557*tilt-14.054 (this is the default Polynomial formula if I'm not mistaken)? 
2. Why are the third numbers in each set of number (prior to *tilt*tilt*tilt) so different (#1 with 9.9453 and #2 with 0.000001272)?

Thank you!

**Comment by thegreatgunbantoad** [comment]
It doesn't matter what the poly is during your readings as you should only be reading the Tilt reading from the iSpindel.  Your gravity should be being read by a hydrometer or other know calibrated external method, not from the iSpindel.

See if this sheet is easier to use: https://github.com/universam1/iSpindel/blob/master/docs/ManualPolyGenerator.xlsx
It uses brackets to ensure the calc is legal and E notation to keep the poly length down.  There used to be an issue with long polys getting cropped.
You'll need to download it btw.

**Reply by thegreatgunbantoad** [reply]
Difficult to answer why your numbers are different without your data and knowing which calculation method you used.

**Reply by higginsg** [reply]
Thank you for the quick reply - I was away from home - apologies for my late reply.

I've attached the readings I took in the spreadsheet and the polynomials it generated.

I'd appreciate it if you'd have a look and tell me if I'm doing something wrong!

I did the [easy calibration method here.](https://www.ispindel.de/docs/Calibration_en.html)

[tilt.xlsx](https://github.com/universam1/iSpindel/files/10323777/tilt.xlsx)

**Comment by thegreatgunbantoad** [comment]
@higginsg 
First thing of note you have cut the 'e-7' out, don't do that it's very much needed.

Edit:
Yeah I think that was pretty much it.  Have a look in here for an alternative format:
[tilt_response.xlsx](https://github.com/universam1/iSpindel/files/10326403/tilt_response.xlsx)

TL:DR:
Use https://github.com/universam1/iSpindel/blob/master/docs/ManualPolyGenerator.xlsx to get:
```
iSpindel 1:	(9.9453E-7*(tilt^3))+(-1.3826E-4*(tilt^2))+(7.9887E-3*tilt)+(8.6720E-1)
iSpindel 2:	(1.2725E-6*(tilt^3))+(-1.7544E-4*(tilt^2))+(9.4635E-3*tilt)+(8.5234E-1)
```
